nixos/modules/config/gtk/gtk.nix

```diff
@@ -0,0 +1,146 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.gtk;
+  gtk2 = cfg.enable && cfg.gtk2;
+
+  toGtk2Ini = key: value:
```

Comment:
This is not an ini file either: https://gitlab.gnome.org/GNOME/gtk/blob/2.24.31/docs/reference/gtk/tmpl/gtkrc.sgml